rgf/utils.py

```diff
@@ -623,6 +623,7 @@ class RGFBinaryClassifierBase(RGFMixin, BaseEstimator, ClassifierMixin):
     def __init__(self, **kwargs):
         for key, value in kwargs.items():
             setattr(self, key, value)
+            setattr(self, '_{0}'.format(key), value)
```

Comment:
.

Reply:
Can you add any test for this feature?

Reply:
What kind of test do you want to see?
Without this change (I'd better call it  *little hack*) everything simply crashes.

Reply:
How about confirm `self._***` exists after construction?